hack/release.sh

```diff
@@ -27,6 +27,13 @@ sed -i "s/\(.*startingCSV.*\)${previous_version}\(.*\)/\1${released_version}\2/g
 echo 'Add updated readme to the source tree'
 git add README.md
 
+echo 'Generating new release for workflow e2e tests'
+cp tests/releases/${released_version}.go tests/releases/${future_version}.go
+sed -i "s/Version: \"${released_version}\",/Version: \"${future_version}\",/" tests/releases/${future_version}.go
+
+echo 'Committing new master release module'
+git add tests/releases/${future_version}.go
```

Comment:
This is really necessary ? playing around with git is dangerous.

Reply:
I feel doing all these steps manually is more dangerous and most of all annoying. It will end up as a PR, so there is still review process involved and it worked until now. Let's hope it won't delete master branch.